enumeratum-core-jvm-tests/src/test/scala/enumeratum/EnumJVMSpec.scala

```diff
@@ -3,39 +3,5 @@ package enumeratum
 import org.scalatest.funspec.AnyFunSpec
 import org.scalatest.matchers.should.Matchers
 
-class EnumJVMSpec extends AnyFunSpec with Matchers {
```

Comment:
Refactor tests based call to compiler (not portable accross Scala version) with [SBT code generation](https://github.com/lloydmeta/enumeratum/pull/349/files#diff-7e310c995c545aa82c4741a740b97408c81874ee2024c7db68da5b85a2be587bR14) (portable and somewhat safer)